programs/autocrat/src/lib.rs

```diff
@@ -41,22 +41,27 @@ pub mod autocrat {
     }
 
     // ==== autocrat
+    // can anyone add instructions to anyone else's `ProposalInstructions` account?
+    // that could be a security problem
     pub fn create_proposal_instructions(ctx: Context<CreateProposalInstructions>, instructions: Vec<ProposalInstruction>) -> Result<()> {
         instructions::autocrat::create_proposal_instructions::handler(ctx, instructions)
     }
-
+    
     pub fn add_proposal_instructions(ctx: Context<AddProposalInstructions>, instructions: Vec<ProposalInstruction>) -> Result<()> {
         instructions::autocrat::add_proposal_instructions::handler(ctx, instructions)
     }
 
+    // why two parts?
```

Comment:
I couldn't get all the accounts to fit in the stack size when trying to do it in one. Maybe there are other solutions to that issue though, open to ideas.